packages/elements/src/chart/elements/chart.ts

```diff
@@ -333,8 +327,29 @@ export class Chart extends BasicElement {
     // set global config again in case the CSS font isn't loaded when updating the chart
     this.setGlobalConfig();
     this.decorateColors(chart);
+    this.decorateGridColors(chart);
   };
 
+  /**
+   * Set grid color in beforeUpdate event from Chartjs
+   * @param chart Chart.js instance
+   * @returns {void}
+   */
+  private decorateGridColors(chart: ChartJS): void {
```

Comment:
There is no unit test for this function, right? Let's exclude this function from test coverage.

Reply:
Added test for this case.